app/components/submission-action-cell/component.js

```diff
@@ -20,6 +23,11 @@ export default class SubmissionActionCell extends Component {
     return this.editable;
   }
 
+  @action
+  toggleDelete() {
+    this.showDeleteModal = !this.showDeleteModal;
+  }
+
   @action
   delete() {
     this.submission.destroyRecord();
```

Comment:
I think this is a little out of band with the purpose of this PR, but I think we should make a note here for ourselves that since `destroyRecord` will kick off an async operation here we may want to make this a task and wrap it in a try / catch.